dropbox/session.py

```diff
@@ -59,8 +51,19 @@ def init_poolmanager(self, connections, maxsize, block=False, **_):
         )
 
 def pinned_session(pool_maxsize=8, ca_certs=None):
-    http_adapter = _SSLAdapter(pool_connections=4, pool_maxsize=pool_maxsize, ca_certs=ca_certs)
+    # always verify, use cert bundle if provided
+
     _session = requests.session()
-    _session.mount('https://', http_adapter)
 
+    # requests
+    if ca_certs is not None:
+        _session.verify = ca_certs
+    else:
+        _session.verify = True
+
+    # urllib3 within requests
+    http_adapter = _SSLAdapter(pool_connections=4, pool_maxsize=pool_maxsize, ca_certs=ca_certs)
```

Comment:
We may as well set that to 6, as that's the current number supported by Chrome.

Reply:
I'd prefer not as it's not really related to the CA bundle changes being made here. We should do that separately IMO.

Reply:
I see, I thought it was new code but maybe the `pool_connections=4` was carried over from somewhere else?